### PR TITLE
!!! [v6r15] almost purely pylint

### PR DESCRIPTION
**Pull request by fstagni**
pilot_1 still in in this one

### CODE REVIEW

**Comment by chaen** [comment]
Review OK

File: FrameworkSystem/Client/MonitoringClient.py [hunk 1]
```diff
@@ -1,9 +1,13 @@
-# $HeadURL$
-__RCSID__ = "b7db10b (2013-03-06 01:10:41 +0100) Andrei Tsaregorodtsev <atsareg@in2p3.fr>"
+""" Exposes singleton object gMonitor, which is instance of MonitoringClient class
+
+    Uses RPC Framework/Monitoring service. Calls registerActivities exposed function
+"""
+
+__RCSID__ = "$Id"
```
Comment (andresailer) [inline FrameworkSystem/Client/MonitoringClient.py]:
```
__RCSID__ = "$Id$"
```

File: WorkloadManagementSystem/Client/JobState/CachedJobState.py [hunk 2]
```diff
@@ -1,5 +1,10 @@
+""" I wish who wrote this would have put some doc...
```
Comment (andresailer) [inline WorkloadManagementSystem/Client/JobState/CachedJobState.py]: Why not leave this empty, until somebody writes a real docstring?

**Comment by atsareg** [comment]
Confilcts, please, update

**Comment by fstagni** [comment]
Resolved

**Comment by atsareg** [comment]
Federico, this PR is somewhat strange, it contains commits which are not yours an which are already merged. This can be if your branch is rebased on vrel-v6r14 for example so that already existing commits are getting new commit IDs and git thinks those commits are new. I do not think this is a problem itself but the history will be quite strange after the mere. Can you please check if you can make a clean PR against rel-v6r15.

**Pull request closed without merging**